src/Network/Network/Cortex/HubVnetConnection/UpdateAzureRmHubVirtualNetworkConnectionCommand.cs

```diff
@@ -127,6 +124,10 @@ public override void Execute()
             {
                 connectionToModify.EnableInternetSecurity = this.EnableInternetSecurity.Value;
             }
+            else
```

Comment:
dont do this, i think the main purpose of this cmdlet is to allow the user to be able to modify the enab;eInternetSecurity value , revert this

Reply:
What I was trying to do was make the option to be true by default if not specified (as you had indicated in our offline discussion). The user would still be able to set it to false if they do so explicitly.

Reply:
option to be true by default if not specified (as you had indicated in our offline discussion). -> this is only for new connections to be created not for existing connections. 
for update scenario : we dont want to modify the flag value in case the cx has forgotten to explicitly mention it.

Reply:
Hmm, why make it asymmetric? If we are doing it for 'New', why not keep it consistent and do it on the 'Update' too? We can talk more offline..